tests/cross/src/test/scala/tests/pc/AutoImplementAbstractMembersSuite.scala

```diff
@@ -78,7 +75,27 @@ class AutoImplementAbstractMembersSuite extends BaseCodeActionSuite {
        |
        |  }
        |}
-       |""".stripMargin
+       |""".stripMargin,
+    compat = Map(
```

Comment:
It would be better to align the number of newlines with the Scala2 version, but it's so complicated :\